Source/SwiftLintBuiltInRules/Rules/Style/ImplicitGetterRule.swift

```diff
@@ -31,7 +31,7 @@ private extension ImplicitGetterRule {
     final class Visitor: ViolationsSyntaxVisitor<ConfigurationType> {
         override func visitPost(_ node: AccessorBlockSyntax) {
             guard let getAccessor = node.getAccessor,
-                  node.setAccessor == nil,
+                  node.accessorsList.count == 1,
```

Comment:
As an optimization, we could do the check for the number of elements first to avoid searching for `get` in `getAccessor`.